cloudstack.go

```diff
@@ -148,6 +149,11 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 			Name:  "cloudstack-network-id",
 			Usage: "CloudStack network id",
 		},
+		mcnflag.IntFlag{
+			Name:  "cloudstack-network-interface",
+			Usage: "CloudStack network interface (force interface to get ip address)",
```

Comment:
Make it clearer that we should provide the index for the interface:

```suggestion
			Usage: "CloudStack network interface index",
```